zapcore/json_encoder_test.go

```diff
@@ -109,6 +109,20 @@ func TestJSONEncodeEntry(t *testing.T) {
 				}),
 			},
 		},
+		{
+			desc: "zero_time_omitted",
+			expected: `{
```

Comment:
Nice. Would you mind adding a similar check to the console_encoder_test.go as well?